### DISCUSSION TITLE
class-exercise

### DISCUSSION THREAD

**Discussion by Debbie-Blaq** [community/community#129631] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

after creating a repository, I couldn't find ''automatic page generator'' under my setting, please help me, because I want to launch automatic page generator

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Debbie-Blaq ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by verduscos** [comment, marked as answer]
Hey @Debbie-Blaq ,

GitHub no longer has an "automatic page generator", but has been replaced by Jekyll which you can find under GitHub Pages. You might find this [discussion](https://github.com/orgs/community/discussions/36779) useful or take a look at the docs [here](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll) for setting up a GitHub page.

**Reply by Debbie-Blaq** [reply]
Thanks so much

**Comment by nafiulhaqueinan** [comment]
GitHub has deprecated this feature and replaced it with GitHub Pages, which allows you to host static websites directly from a GitHub repository.
